style.css

```diff
@@ -0,0 +1,356 @@
+*
+    {
+        font-family: Arial, Helvetica, sans-serif;
+        color: white;
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+    }
+    
+    body
+    {
+        background-color: #181818;
+        margin: 0 auto;
```

Comment:
Try not to have a margin on the body. You should use things like `display: flex;`, `justify-content: center;` and `flex-direction: center;`.